apps/pie-storybook/stories/pie-tag.stories.ts

```diff
@@ -73,21 +81,22 @@ const Template : TemplateFunction<TagProps> = ({
     variant,
     size,
     isStrong,
+    isDimmed,
     showIcon,
     slot,
 }) => html`
     <pie-tag
         variant="${variant}"
         size="${size}"
-        ?isStrong="${isStrong}">
+        ?isStrong="${isStrong}"
+        ?isDimmed="${isDimmed}">
         ${showIcon ? html`<icon-heart-filled slot="icon"></icon-heart-filled>` : nothing}
         ${slot}
     </pie-tag>
 `;
 
 const createTagStory = createStory<TagProps>(Template, defaultArgs);
 
-export const Default = createTagStory();
```

Comment:
do we no longer want a default?

Reply:
we don't have a variant called default, it's called neutral.